src/Layouts/Default/components/SelectLocale/index.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { useI18n } from "providers/I18n"
+import { css } from "emotion"
+import elLogo from "../../../../utils/lang_images/greece.png"
+import enLogo from "../../../../utils/lang_images/uk.png"
```

Comment:
Let's place those images under the existing folder `src/images`. `utils` is not the place for images

Reply:
also let's name the files like our locales. el.png, en.png